<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResearcherWindowClass</class>
 <widget class="QMainWindow" name="ResearcherWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResearcherWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="ideasTableView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Title : </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="descrLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Duration: </string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="durLineEdit"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Accept</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Develop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>Save All</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ResearcherWindowClass</receiver>
   <slot>addIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ResearcherWindowClass</receiver>
   <slot>acceptIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ResearcherWindowClass</receiver>
   <slot>develop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>ResearcherWindowClass</receiver>
   <slot>saveIdea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>ResearcherWindowClass</receiver>
   <slot>saveAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addIdea()</slot>
  <slot>acceptIdea()</slot>
  <slot>develop()</slot>
  <slot>saveIdea()</slot>
  <slot>saveAll()</slot>
 </slots>
</ui>
